t/plugin/limit-count-redis-cluster.t

```diff
@@ -63,7 +63,7 @@ __DATA__
 GET /t
 --- error_code: 400
 --- response_body
-{"error_msg":"failed to check the configuration of plugin limit-count err: failed to validate dependent schema for \"policy\": value should match only one schema, but matches none"}
+{"error_msg":"failed to check the configuration of plugin limit-count err: else clause did not match"}
```

Comment:
The error messages is confusing after changes, it cannot help API callers troubleshoot effectively, will we have some enhancements?

Reply:
Sure. We can improve it in the jsonschema library.